data_structures/binary_tree/binary_search_tree.py

```diff
@@ -141,6 +141,28 @@ def traversal_tree(self, traversal_function=None):
         else:
             return traversal_function(self.root)
 
+    def inorder(self, arr, Node):
```

Comment:
Type hints are https://docs.python.org/3/library/typing.html

https://docs.python.org/3/library/doctest.html are also missing.

Reply:
I am adding the type hints